src/index.js

```diff
@@ -12,7 +12,45 @@ if (!app.requestSingleInstanceLock()) {
   )
 
   // No windows were opened at this time so we don't need to do app.quit()
-  process.exit(0)
+  process.exit(1)
+}
+
+const issueTemplate = (e) => `Please describe what you were doing when this error happened.
+
+**Specifications**
+
+- **OS**: ${process.platform}
+- **IPFS Desktop Version**: ${app.getVersion()}
```

Comment:
Something of note, `app.getVersion` returns electron version instead of one from `package.json` field:
> ![2018-11-22--18-33-09](https://user-images.githubusercontent.com/157609/48917032-15e21800-ee85-11e8-87bf-4ef4df831bdc.png)

It seems to be a known issue (https://github.com/electron/electron/issues/7085) when running development version from a subdirectory (`./src/`), and should be fine in release build.

Reply:
@lidel building by hand, i.e., `npm run build` will give you the correct value. Only if you run it directly with `npm start` you'll be given Electron's version. We could though read package.json file directly. What do you think?

Reply:
@lidel I'm merging this now. We can take a look at that detail later.